src/amp-story-player/amp-story-player-impl.js

```diff
@@ -508,6 +538,13 @@ export class AmpStoryPlayer {
             );
           });
 
+          const playerConfig = this.readPlayerConfig_();
```

Comment:
Should we parse the config in the buildCallback and then just use `this.playerConfig_`?